launch/docker.go

```diff
@@ -122,7 +122,13 @@ func (d *docker) runBuild(buildEntry buildEntry) error {
 	}
 
 	dockerCommandArgs := []string{"container", "run"}
-	dockerCommandOptions := []string{"--rm", "-v", srcVol, "-v", artVol, "-v", binVol, "-v", habVol, buildImage}
+	sockFile := os.Getenv("SSH_AUTH_SOCK")
+	_, err = os.Stat("/run/host-services/ssh-auth.sock")
```

Comment:
FYI: This file exists in "Docker Desktop VM", not on macOS itself. So, I think there are no meanings to stat on this situation.

Reply:
Hmm, so this isn't working then.